lib/data/dir/quizzes.json

```diff
@@ -5,41 +5,524 @@
       "questions":
       [
         {
-          "text": "What is the cost of domestic violence?",
+          "text": "What is the cost of intimate partner violence exceeds how much per year?",
```

Comment:
Grammar here is slightly off-- maybe "The cost of intimate partner violence exceeds how much per year?"